csro/oiwfs-adc-assembly/publish-model.conf

```diff
@@ -309,8 +316,14 @@ publish {
                     <ul>
                         <li><b>Major:</b> Execute the datum command. </li>
                     </ul>)"""
-      severity = major
-      archive = true
+      // TODO FIXME: The fields below were automatically added and should be updated manually
```

Comment:
updated manually by whom? is it worth making those updates directly in this PR before merging?

Reply:
I think you're right, we should review all of these as part of the PR (and then remove the comments). I'll take a look at this.

Reply:
OK, I reviewed and fixed up the TODO comments in alarms for the CSRO assemblies. However, I think @tnakamot and @jimansohn will have to deal with their own assemblies as I don't know them well enough to guess at how the alarms should be formatted.